src/components/MapView/GroundstationLayers/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders as expected 1`] = `
```

Comment:
I made this comment in your other PR too - I don't think these types of tests are worth having. The test should either be re-written to actually render something that tests the layer code, or we should just get rid of it - it's just adding extra complexity without extra value as is here.

(Unless the specific goal is to test whether the layer handles an empty dataset properly)